src/main/java/org/primefaces/component/datatable/DataTableBase.java

```diff
@@ -23,14 +23,13 @@
  */
 package org.primefaces.component.datatable;
 
-import java.util.Collections;
-import java.util.Map;
+import org.primefaces.component.api.*;
+import org.primefaces.model.FilterMeta;
 
 import javax.el.MethodExpression;
 import javax.faces.component.behavior.ClientBehaviorHolder;
-
-import org.primefaces.component.api.*;
-import org.primefaces.model.FilterMeta;
+import java.util.Collections;
+import java.util.List;
```

Comment:
Typically our imports are java, javax, org, com.  Or at least that is how I have mine set.

Reply:
I think this is a topic that has been discussed in the past but no answer given. I'm using default settings from intellij

Reply:
Understood. You are right I am not sure there is a standard but we should proabbly make one.  I know in the past its more than 5 imports becomes `java.util.*` is the one standard but we should probably document it.  Possibly even use Spotless plugin to correct it.